docs/commands/debug-piral.md

```diff
@@ -143,15 +143,15 @@ Automatically installs missing Node.js packages.
 Also includes the node modules for target transpilation.
 
 - Type: `boolean`
-- Default: `false`
+- Default: `true`
```

Comment:
Should be `false`. Don't know why this was changed here; same is true for the others.

Reply:
Fixed. Docgen has been performed before bootstrapping updates from upstream. Sorry.